bundles/org.openhab.binding.astro/README.md

```diff
@@ -130,6 +130,12 @@ OR
 
 sunset is 22:10 but `latest` is set to 20:00 so the event/datetime value is moved 20:00.
 
+**Force event:** For each trigger channel and `start`, `end` datetime value, you can force the `earliest`, `latest` time of the day, when the event is actually not taking place (e.g. astronomic dawn during summer in Sweden)
+e.g `sun#astroDawn earliest=8:00, latest=20:00 forceEvent=true`
+
+astronomic dawn start is null but `earliest` is set to 06:00 so the event/datetime value is set to 06:00.
```

Comment:
Can you consistent between the example and the explanation, that is either 6 or 8.

Reply:
:-/